autometa/common/external/work_queue.py

```diff
@@ -179,11 +179,11 @@ def main(args):
     parser.add_argument('password', help='</path/to/work-queue/master/password.txt>')
     parser.add_argument('project-name', help='<unique project identifier>')
     # TODO: Allow argparse to handle range of ints for port
-    #parser.add_argument('--port', help="""
-    #The port number to listen on. If zero, then a random port is chosen. A range
-    # of possible ports (low, hight) can be also specified instead of a single integer.
-    #""",
-    #    default=wq.WORK_QUEUE_DEFAULT_PORT, type=int)
+    parser.add_argument('--port', help="""
+    The port number to listen on. If zero, then a random port is chosen. A range
+     of possible ports (low, hight) can be also specified instead of a single integer.
+    """,
+        default="wq.WORK_QUEUE_DEFAULT_PORT", type=int)
```

Comment:
This should not be a string. When you write this out within the temporary file, you may convert it to a string.

Reply:
I am not quite sure how to do this. Perhaps the way to do this is similar to doing [this](https://github.com/KwanLab/Autometa/pull/45#discussion_r407855390)

Reply:
Yes, both will be similar in regards to the routine. However, what you are replacing will be slightly different.